xmodule/video_block/video_block.py

```diff
@@ -152,7 +152,6 @@ class VideoBlock(
     js_module_name = "TabsEditingDescriptor"
 
     uses_xmodule_styles_setup = True
-    requires_per_student_anonymous_id = True
```

Comment:
The `VideoBlock` does not use the anonymous student ID anywhere. The only usage of the `user` service is for requesting the `ATTR_KEY_REQUEST_COUNTRY_CODE`.